packages/website/src/bin/generate-pages.js

```diff
@@ -62,7 +69,16 @@ module.exports = async ({
   fs.writeFileSync(
     metaPath,
     JSON.stringify(
-      { siteName, packagesDescription, links, ...rest },
+      {
+        siteName,
+        packages: {
```

Comment:
Mild consistency thing - why is this the only object being assembled inline in the stringify function? If everything else is defined above, it would be good to define this above as well.